docker/docker-compose.yml

```diff
@@ -0,0 +1,371 @@
+    hostname: edgex-core-command
+    networks:
+      - edgex-network
+    volumes:
+      - db-data:/data/db
+      - log-data:/edgex/logs
+      - consul-config:/consul/config
+      - consul-data:/consul/data
+    depends_on:
+      - metadata
+
+# scheduler container does not come up on Consul - bug fix in the works.
+  scheduler:
+    image: edgexfoundry/docker-support-scheduler
+    ports:
+      - "48085:48085"
+    container_name: edgex-support-scheduler
+    hostname: edgex-support-scheduler
+    networks:
+      - edgex-network
+    volumes:
+      - db-data:/data/db
+      - log-data:/edgex/logs
+      - consul-config:/consul/config
+      - consul-data:/consul/data
+    depends_on:
+      - metadata
+
+  export-client:
+    image: edgexfoundry/docker-export-client
+    ports:
+      - "48071:48071"
+    container_name: edgex-export-client
+    hostname: edgex-export-client
+    networks:
+      - edgex-network
+    volumes:
+      - db-data:/data/db
+      - log-data:/edgex/logs
+      - consul-config:/consul/config
+      - consul-data:/consul/data
+    depends_on:
+      - data
+    environment:
+      - EXPORT_CLIENT_MONGO_URL=edgex-mongo
+      - EXPORT_CLIENT_DISTRO_HOST=export-distro
+      - EXPORT_CLIENT_CONSUL_HOST=edgex-config-seed
+
+  export-distro:
+    image: edgexfoundry/docker-export-distro
+    ports:
+      - "48070:48070"
+    container_name: edgex-export-distro
+    hostname: edgex-export-distro
+    networks:
+      - edgex-network
+    volumes:
+      - db-data:/data/db
+      - log-data:/edgex/logs
+      - consul-config:/consul/config
+      - consul-data:/consul/data
+    depends_on:
+      - export-client
+    environment:
+      - EXPORT_DISTRO_CLIENT_HOST=export-client
+      - EXPORT_DISTRO_DATA_HOST=edgex-core-data
+      - EXPORT_DISTRO_CONSUL_HOST=edgex-config-seed
+      - EXPORT_DISTRO_MQTTS_CERT_FILE=none
+      - EXPORT_DISTRO_MQTTS_KEY_FILE=none
+
+  rulesengine:
+    image: edgexfoundry/docker-support-rulesengine
+    ports:
+      - "48075:48075"
+    container_name: edgex-support-rulesengine
+    hostname: edgex-support-rulesengine
+    networks:
+      - edgex-network
+    volumes:
+      - db-data:/data/db
+      - log-data:/edgex/logs
+      - consul-config:/consul/config
+      - consul-data:/consul/data
+    depends_on:
+      - export-distro
+
+#################################################################
+# Device Services
+#################################################################
+
+  device-virtual:
+    image: edgexfoundry/docker-device-virtual
+    ports:
+      - "49990:49990"
+    container_name: edgex-device-virtual
+    hostname: edgex-device-virtual
+    networks:
+      - edgex-network
+    volumes:
+      - db-data:/data/db
+      - log-data:/edgex/logs
+      - consul-config:/consul/config
+      - consul-data:/consul/data
+    depends_on:
+      - data
+      - command
+      - scheduler
+      - notifications
+
+#   device-bluetooth:
```

Comment:
I do not like the idea of dragging this dead code around. It's in the Git, if someone needs it later, can be always checked.

Reply:
I will remove them.